contxt/services/auth.py

```diff
@@ -12,9 +12,18 @@ class AuthService(ConfiguredApi):
             base_url="https://contxtauth.com/v1",
             client_id="75wT048QcpE7ujwBJPPjr263eTHl4gEX",
         ),
+        ApiEnvironment(
+            name="staging",
+            base_url="https://contxt-auth-service.staging.ndustrial.io/v1",
+            client_id="7TceUsM1eC4nKmdoC717383DWyfc9QoY",
+        ),
     )
 
-    def __init__(self, env: str = "production", **kwargs) -> None:
+    def __init__(
+        self,
+        env: str = "production",
+        **kwargs,
```

Comment:
fyi, removing this comma will tell black black to collapse this back down to a single line (see [magic trailing comma](https://black.readthedocs.io/en/stable/the_black_code_style.html?highlight=comma#the-magic-trailing-comma))
```suggestion
        **kwargs
```